src/javascript/app/pages/mb_trade/mb_event.js

```diff
@@ -106,20 +108,19 @@ const MBTradingEvents = (() => {
         }
 
         const validatePayout = (payout_amount, $error_wrapper) => {
-            const contract            = MBContract.getCurrentContracts();
-            const min_amount          = is_jp_client ? 1 : 0;
-            const max_contract_amount = Array.isArray(contract) && contract.length && contract[0].expiry_type !== 'intraday' ? 20000 : 5000;
-            const max_client_amount   = is_jp_client ? 100 : max_contract_amount;
+            const market              = MBSymbols.getAllSymbols()[MBDefaults.get('underlying')].market;
+            const selected_currency   = MBDefaults.get('currency');
+            const max_client_amount   = State.getResponse(`landing_company.financial_company.currency_config.${market}.${selected_currency}.max_payout`) || 5000;
 
             let is_valid  = true;
             let error_msg = '';
 
             if (!payout_amount || isNaN(payout_amount)) {
                 is_valid  = false;
                 error_msg = localize('Should be a valid number.');
-            } else if (+payout_amount < min_amount || +payout_amount > max_client_amount) {
+            } else if (+payout_amount > max_client_amount) {
                 is_valid  = false;
-                error_msg = localize('Should be between [_1] and [_2]', [min_amount, max_client_amount]);
+                error_msg = localize('Should be below [_1]', [max_client_amount]);
```

Comment:
can we use `Should be less than [_1]` since we already have translation string for it